sdk/src/transaction_context.rs

```diff
@@ -999,7 +999,7 @@ impl<'a> BorrowedAccount<'a> {
     /// Returns whether this account is executable (transaction wide)
     #[inline]
     pub fn is_executable(&self) -> bool {
-        self.account.executable()
+        is_builtin(&self.account) || is_executable(&self.account)
```

Comment:
Asymmetry happens because there are two different usages of `executable`, each one has its own meaning.

The first is in the transaction loading, happens in `bank.rs`. In this usage, we need to differentiate between `is_builtin` and `not-builtin- but-executable` - they are treated differently.

The other is in program runtime, which checks executable on "borrowed_account". Here the meaning of `executable` includes both `is_builtin` and `not-builtin- but-executable`.